src/components/badges/index.ts

```diff
@@ -1,5 +1,7 @@
-import Badges from './Badges';
```

Comment:
this will not work (or it works differently than intended) 

To be consistent we should always structure each component in its own folder, so that there is only 1 default export (the actual component) and several member exports (for types and stuff).